### DISCUSSION TITLE
Deleting a record in NetSuite

### DISCUSSION THREAD

**Discussion by darren-kimberbell** [netsuitephp/netsuite-php#141] [Q&A]
I have my create and update functions working using the PHP toolkit, but I am having trouble deleting a record in NetSuite via PHP.

Does anyone have any example PHP code to that they would be willing to share to delete a record in netsuite?

In this case I am trying to delete a CustomRecord, but any code would be useful.  I see the DeleteRequest requires a baseRef object.  I can provide the baseRef object for the CustomRecord, but it doesn't have a value for internalId, and so I am unsure how to tell it which record to delete.

Any help to point me in the right direction would be appreciated.  Thank you!

Darren Nay

**Comment by jrebs** [comment]
Just off the top of my head, can you use an object of `NetSuite\Classes\RecordRef` in place of the BaseRef? It extends that and does define an internalId.
